lib/bolt/config/transport/docker.rb

```diff
@@ -15,7 +15,7 @@ class Docker < Base
           shell-command
           tmpdir
           tty
-        ].freeze
+        ].concat(RUN_AS_OPTIONS).sort.freeze
```

Comment:
The inventory schema needs to be regenerated to include these options. Looks like the CI job didn't get triggered since the paths don't include `lib/bolt/transport/**`. 😬

Reply:
Oh good call, updated the trigger for that workflow as well.